src/js/store/orgs/actions.ts

```diff
@@ -81,24 +109,16 @@ export const provisionOrg = ({
   const state = getState();
   /* istanbul ignore else */
   if (isCurrentUser(originating_user_id, state)) {
-    const task = selectTaskById(state, model.task);
-    let msg = {
-      [ORG_TYPES.DEV]: i18n.t('Successfully created Dev Org.'),
-      [ORG_TYPES.QA]: i18n.t('Successfully created Test Org.'),
-    };
-    if (task) {
-      msg = {
-        [ORG_TYPES.DEV]: `${i18n.t('Successfully created Dev Org for task')} “${
-          task.name
-        }”.`,
-        [ORG_TYPES.QA]: `${i18n.t('Successfully created Test Org for task')} “${
-          task.name
-        }”.`,
-      };
+    const MESSAGES = getOrgMessages();
+    let msg = MESSAGES.CREATED[model.org_type];
+    const { name, type } = getOrgParent(model, state);
+    /* istanbul ignore else */
+    if (name && type) {
+      msg = `${MESSAGES.CREATED[type][model.org_type]} “${name}”.`;
```

Comment:
Avoid concatenating translated strings like this. The name might not belong at the end in all languages -- think about how languages like German sometimes put the verb at the end.

I think we probably want strings like "Successfully created {org} for {context}" and then separately translate the org type and context to pass in to the overall translation.